common/static/common/js/spec/utils/edx.utils.validate_spec.js

```diff
@@ -0,0 +1,197 @@
+;(function (define) {
+    'use strict';
+    define(['jquery', 'common/js/utils/edx.utils.validate'],
```

Comment:
Since this is unfortunately showing as an add/delete instead of a move, I will comment on the changes.